src/executables/util_cs.c

```diff
@@ -4544,18 +4545,126 @@ flashback (UTIL_FUNCTION_ARG * arg)
 int
 memmon (UTIL_FUNCTION_ARG * arg)
 {
+#if defined(CS_MODE)
   UTIL_ARG_MAP *arg_map = arg->arg_map;
-  char *module;
-  bool transaction, show_all;
+  char er_msg_file[PATH_MAX];
+  const char *database_name;
+  char *module, *stop;
+  bool transaction, show_all, need_shutdown;
   int tran_count;
+  int module_index = MMON_MODULE_LAST;
 
   module = utility_get_option_string_value (arg_map, MEMMON_MODULE_S, 0);
   transaction = utility_get_option_bool_value (arg_map, MEMMON_TRANSACTION_S);
   tran_count = utility_get_option_int_value (arg_map, MEMMON_TRAN_COUNT_S);
   show_all = utility_get_option_bool_value (arg_map, MEMMON_SHOW_ALL_S);
 
+  database_name = utility_get_option_string_value (arg_map, OPTION_STRING_TABLE, 0);
+  if (database_name == NULL)
+    {
+      goto print_memmon_usage;
+    }
+
+  /* error check phase */
+  if (check_database_name (database_name))
+    {
+      goto error_exit;
+    }
+
+  if ((!transaction) && tran_count)
```

Comment:
It looks working different behavior:
- cubrid memmon --tran-count=0 -> normal execution
- cubrid memmon --tran-count=1 -> error

Reply:
I fix the input of  tran_count option to string because we have to check the tran_count value whether user entered or not(default value).